README.rst

```diff
@@ -19,7 +18,7 @@ The documentation has five sections:
 - `Getting started <https://rocky.docs.pyansys.com/version/stable/getting_started/index.html>`_: Learn
   how to install PyRocky in user mode and then launch it.
 - `User guide <https://rocky.docs.pyansys.com/version/stable/user_guide/index.html>`_: Understand how to
-  use the Rocky PrePost API with PyRocky.
+  use the Rocky PrePost Scripting with PyRocky.
```

Comment:
```suggestion
  use Rocky PrePost Scripting with PyRocky.
```
Is "Scripting" capitalized because it is part of a proper noun? If not, use a lower case "s" instead.

Reply:
_PrePost Scripting_ is the name of the feature in which you can automate Rocky using scripts.